File: 1848-minimum-distance-to-the-target-element/1848-minimum-distance-to-the-target-element.java
class Solution {
    public int getMinDistance(int[] nums, int target, int start) {
        
        int min=Integer.MAX_VALUE;
        for(int i=0;i<nums.length;i++){
            int num=0;
            if(nums[i]==target){
                num=num+Math.abs(i-start);
                if(min>num){
                    min=num;
                }
            }
        }
        return min;
    }
}

File: 1848-minimum-distance-to-the-target-element/1848-minimum-distance-to-the-target-element.cpp
class Solution {
public:
    int getMinDistance(vector<int>& nums, int target, int start) {
        int min=INT_MAX;
        for(int i=0;i<nums.size();i++){
            int num;
            if(target==nums[i]){
              num=abs(i-start);
                if(min>num){
                  min=num;
                }
            }
        }
        return min;
    }
};